### PR TITLE
Upgrade base images from ubi8 to ubi9

### PR DESCRIPTION
**Pull request by Roming22**
This fixes a vulnerability on python-pip introduced by the git package.

Signed-off-by: Romain Arnaud <rarnaud@redhat.com>

### CODE REVIEW

File: operator/images/cluster-setup/Dockerfile [hunk 1]
```diff
@@ -17,17 +17,21 @@ LABEL build-date= \
...
-RUN mkdir $HOME
-VOLUME /workspace
-WORKDIR /workspace
-ENTRYPOINT ["/opt/cluster-setup/bin/install.sh"]
+# USER 65532:65532
+# ENV HOME /tmp/home
+# RUN mkdir $HOME
+# VOLUME /workspace
+# WORKDIR /workspace
+# ENTRYPOINT ["/opt/cluster-setup/bin/install.sh"]
```
Comment (xinredhat) [inline operator/images/cluster-setup/Dockerfile:37]: I guess we should uncomment above commands, right?

**Comment by bnallapeta** [comment]
@Roming22 Needs rebase after the merge of 396.

**Comment by bnallapeta** [comment]
@Roming22 Isn't this [image](https://github.com/openshift-pipelines/pipeline-service/tree/main/developer/images/dependencies-update) supposed to upgrade the base image for us automatically?

File: operator/images/cluster-setup/Dockerfile (hunk 1 above)
Comment (Roming22) [reply operator/images/cluster-setup/Dockerfile]: Fixed

**Comment by Roming22** [comment]
> @Roming22 Isn't this [image](https://github.com/openshift-pipelines/pipeline-service/tree/main/developer/images/dependencies-update) supposed to upgrade the base image for us automatically?

It updates the SHA of the reference image, not the reference images themselves (i.e. it updates the `FROM` lines, but not the `#@FROM` lines).

**Comment by bnallapeta** [comment]
> It updates the SHA of the reference image, not the reference images themselves (i.e. it updates the FROM lines, but not the #@FROM lines).

Got it. Thank you.

**Pull request merged**